docker-compose.yml

```
@@ -0,0 +1,37 @@
+version: "3"
+services:
+  mongo:
+    image: mongo
+    container_name: mongodb
+    ports:
+      - "27017:27017"
+    command: mongod
+    volumes:
+      - "~/data:/data/db"
```

I have some data folder for other projects, it's better to put it inside the project (also adding it to .gitignore)
```suggestion
      - "./data:/data/db"
```

---

Yes, good suggestion, I am currently testing it locally